back/engines/commercial/moderation/app/models/moderation/moderation.rb

```diff
@@ -4,6 +4,7 @@ class Moderation < ActiveRecord::Base
     self.primary_key = 'id'
 
     has_one :moderation_status, foreign_key: :moderatable_id, foreign_type: :moderatable_type
+    has_one :inappropriate_content_flag, foreign_key: :flaggable_id, foreign_type: :flaggable_type, class_name: 'FlagInappropriateContent::InappropriateContentFlag' # TODO patch
```

Comment:
Stays here for now?

Reply:
Well spotted! Didn't see this, so I will patch it :) In this case I think it makes the most sense to patch it in the flagging engine because the flagging feature requires the moderation feature to work. But otherwise, I wonder if it makes sense for one engine to patch another one, as this only reverses the direction of the dependency, while making the code more complex.